configuration/themes.md

```diff
@@ -5,53 +5,76 @@ description: Zed UI and syntax themes explained
 # Themes
 
 {% hint style="warning" %}
-Themes in Zed are heavily a work in progress. Themes are not guaranteed to stay the same or stay in the app as we iterate on the theme system.
+Zed themes and their underlying systems are undergoing significant development. They should not be considered stable or final. Note that themes are subject to change or removal as the theme system moves towards a stable version.
 {% endhint %}
 
-The Theme currently refers to both the UI theme and the syntax theme. These are currently linked, but in the future will likely be separated or individually customizable.
+A Zed Theme refers to and effects both the UI and the syntax highlighting of Zed.
 
-### Known issues
+We will not have formal UI and syntax themes the like Atom, but will provide a way to override syntax styles (and more) from a single theme.
```

Comment:
```suggestion
We will not have separate UI and syntax themes the like Atom, but will provide a way to override syntax styles (and more) from a single theme.
```

"separate" seems like it's a better fit for this ("formal" confused me at first)